specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+        "operationId": "ServerMetrics_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "name",
+            "description": "Unique name for server metrics, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Requested server metrics",
+            "schema": {
+              "$ref": "#/definitions/ServerMetricsModel"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get server metrics by an unique name": {
+            "$ref": "./examples/GetServerMetrics.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ServerMetrics"
+        ],
+        "summary": "Delete server metrics configuration by its name",
+        "operationId": "ServerMetrics_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "name",
+            "description": "Unique name for server metrics, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The server metrics configuration is deleted."
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a server metrics configuration": {
+            "$ref": "./examples/DeleteServerMetrics.json"
+          }
+        }
+      }
+    },
+    "/serverMetricsConfig": {
```

Comment:
As with app-components, I think serverMetricsConfig (which should be kebab-case "server-metrics-config" when in the path) should either be simply embedded into the Test and TestRuns schemas or a sub-path of /loadtests/{testId} and /testruns/{testRunId}.

Reply:
changed to kebab case